docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,38 @@
+# # Quick start
+
+## What you'll do
+
+- Deploy the charm.
+- Provide GitHub runners on OpenStack mode with images.
```

Comment:
Provide GitHub runners on OpenStack mode with images.? I was expecting to "build GitHub runner images"

Reply:
It provides the image id over the relation for GitHub runners - renamed it to "Integrate with GitHub runners"

Reply:
Addressed all the comments, thank you for the review!